### ISSUE TITLE
Default master coordinates cannot hit zero while normalized

### ISSUE THREAD

**Issue by yarmola** [fonttools/fonttools#1631]
I have an issue with rounding the default master location when axis map is present and that map does not have a point for the default master. I know that's wrong, but I don't think current designSpace doc requires to map default location.

Perfectly integer design coordinate of the default master maps to some floating-point user coordinate which then is recorded (in slightly rounded form, 4 points after dot) as axis default.

It will never get back to normalized zero, which will make this piece of code fail:

<img width="461" alt="Screenshot 2019-06-03 at 03 21 24" src="https://user-images.githubusercontent.com/407038/58769450-d755b480-85af-11e9-8609-abe684e3ca0a.png">

(which is what is happening here)

I will add a work-around (by inserting "rounded" default location to the map), but I think it will be good to 1) add a point to the designSpace document asking to map default location and 2) maybe replace that  == 0 with something that is more floating-point compatible, so it will tolerate "almost zero values" (0.0001213, etc.).

**Comment by anthrotype** [comment]
I think we should update the doc to say that a mapping for the default master (as well as for minimum and maximum) is required if an axis has any map elements.

can you show what is your designspace axes map configuration?

I don't thin we need to do any rounding. A default master is currently required to be present in a DesignSpace to build a varfont, and that master needs to be given an explicit mapping if for a given axis the user vs design locations are different.

**Comment by yarmola** [comment]
I agree that designSpace spec should include this addition (which is very logical, as "implied" map contains min, def, max values by definition). 

My current fix is doing exactly that: adding the point for the default master to the map. It solves the problem.
